stable/kommander/templates/grafana/opsportal-credentials-secret.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.grafana.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: copy-opsportal-credentials-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "kommander.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    metadata:
+      name: copy-opsportal-credentials-secret
+    spec:
+      containers:
+        - name: kubectl
+          # --export flag is deprecated so we need to stick with this kubectl version
+          image: bitnami/kubectl:1.16.2
+          imagePullPolicy: IfNotPresent
+          command:
+            - /bin/sh
+            - -c
+            - kubectl get secret ops-portal-credentials --namespace=kubeaddons --export -o yaml | kubectl apply --namespace={{ .Release.Namespace }} -f -
```

Comment:
@jr0d @jongiddy I also just want to make sure it's okay to be copying the `ops-portal-credentials` secret from `kubeaddons` ns to `kommander` ns. The secret is needed to get the user to pass into the `X-Forwarded-User` header for Grafana requests

Reply:
@gracedo it does allow anyone with secret view privileges in the kommander namespace access to the ops portal password; which they wouldn't have had before. While more it's more difficult, perhaps consider only copying the username. 

What would happen access for the ops-portal user was disabled?

Reply:
Made the change to only copy over the username https://github.com/mesosphere/charts/pull/386/commits/0a2615e24b4cb259f3df350992544b656073ad17